EIPS/eip-1.md

```diff
@@ -103,9 +93,11 @@ Each EIP must begin with an RFC 822 style header preamble, preceded and followed
 
 ` * discussions-to:` <url>
 
-` status:` <Draft | Active | Accepted | Deferred | Rejected | Withdrawn | Final | Superseded>
+` status:` <Draft | Last Call | Approved | Final | Active | Deferred | Rejected | Superseded>
+
+`* review-period:` Starting YYYY-MM-DD, ending YYYY-MM-DD
```

Comment:
This should be just a single date - either start or end - in order to be human-readable.

Reply:
Updated in 0ebde39